Source/Core/InputCommon/ControllerEmu/ControlGroup/IMUGyroscope.h

```diff
@@ -33,14 +34,16 @@ class IMUGyroscope : public ControlGroup
 private:
   using Clock = std::chrono::steady_clock;
 
-  void RestartCalibration() const;
-  void UpdateCalibration(const StateData&) const;
+  bool AreInputsBound() const;
+  bool CanCalibrate() const;
+  void RestartCalibration();
+  void UpdateCalibration(const StateData&);
 
   SettingValue<double> m_deadzone_setting;
   SettingValue<double> m_calibration_period_setting;
 
-  mutable StateData m_calibration = {};
```

Comment:
Just think it's nicer to have `GetState(bool update)` instead than `GetState() const` with mutable stuff, which does not make it clear externally that GetState() is infact an update.